docs/contributing/developing.md

```diff
@@ -28,6 +28,14 @@ cd wagtail
 
 **With your preferred [virtualenv activated](virtual_environment_creation),** install the Wagtail package in development mode with the included testing and documentation dependencies:
 
+### For PowerShell/CMD (Windows)
```

Comment:
No need to be this explicit with sub-headings, and let's try to align with the documentation approach using `doscon`. https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#when-using-console-terminal-code-blocks

Keep the MacOs/Linux one first without any sub-title and then have a small note before showing the Windows equivalent.

Something like this...

````

```sh
pip install -e ."[testing,docs]" --config-settings editable-mode=strict -U
```

Or on Windows

```doscon
pip install -e .[testing,docs] --config-settings editable-mode=strict -U
```

````